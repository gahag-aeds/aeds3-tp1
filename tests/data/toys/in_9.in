10
0.0000 0.44697 0.81885 0.43575 0.97177 0.33669 0.12267 0.30434 0.62428 0.42659 
0.55303 0.0000 0.13466 0.14839 0.28317 0.12246 0.09326 0.22451 0.3934 0.18448 
0.18115 0.86534 0.0000 0.47067 0.03307 0.11099 0.29313 0.42909 0.37075 0.88 
0.56425 0.85161 0.52933 0.0000 0.95578 0.39902 0.7071 0.77699 0.19118 0.72909 
0.02823 0.71683 0.96693 0.04422 0.0000 0.50685 0.35117 0.88341 0.53987 0.13233 
0.66331 0.87754 0.88901 0.60098 0.49315 0.0000 0.28633 0.67956 0.51153 0.46212 
0.87733 0.90674 0.70687 0.2929 0.64883 0.71367 0.0000 0.17419 0.44452 0.50384 
0.69566 0.77549 0.57091 0.22301 0.11659 0.32044 0.82581 0.0000 0.2858 0.64716 
0.37572 0.6066 0.62925 0.80882 0.46013 0.48847 0.55548 0.7142 0.0000 0.47535 
0.57341 0.81552 0.12 0.27091 0.86767 0.53788 0.49616 0.35284 0.52465 0.0000 
