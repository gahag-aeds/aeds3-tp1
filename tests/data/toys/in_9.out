0.085566 0.015336 0.046123 0.177353 0.060570 0.153512 0.135877 0.083447 0.149077 0.093141 